keep/api/api.py

```diff
@@ -159,25 +160,25 @@ def get_app(
     app.include_router(rules.router, prefix="/rules", tags=["rules"])
     app.include_router(preset.router, prefix="/preset", tags=["preset"])
     app.include_router(groups.router, prefix="/groups", tags=["groups"])
+    app.include_router(users.router, prefix="/users", tags=["users"])
 
     # if its single tenant with authentication, add signin endpoint
     logger.info(f"Starting Keep with authentication type: {AUTH_TYPE}")
     # If we run Keep with SINGLE_TENANT auth type, we want to add the signin endpoint
-    if AUTH_TYPE == AuthenticationType.SINGLE_TENANT.value:
-
+    if AUTH_TYPE == "SINGLE_TENANT":
```

Comment:
Why not use the enum?

Reply:
was debugging the missing jwt and tried to see if it was due to enum failure... (before i remembered to add the env)
forgot to restore...